Update PlayersManual.md (#74)
endless-sky/endless-sky-wiki: wiki/PlayersManual.md

:--- | :---
Buy | +, Return, or Space
Sell | -, Delete, or Backspace
Buy / sell all | Shift + B / S
View commodity price map | Map key

Multipliers: | &nbsp;
:--- | :---
5x | Shift
20x | Control / Command

> 500x | Option / Alt

Reviewer: ```suggestion
100x | Command & Shift
500x | Option / Alt
```
Something like this as well, maybe?

Author: Yeah, except my Windows/Linux keyboard doesn't have a "Command" key... could/should we add a Mac/PC column?

Reviewer: I'm not sure that would be necessary. How about this?
```suggestion
100x | Command / Control & Shift
500x | Option / Alt
```

Author: Now I feel like it needs parentheses to clarify the order of operations...
I'm not looking at the table right now I'm only looking at the Delta in the code so maybe it looks better in the table but this could be interpeeted as (command) or (control+shift), or (command or control) + shift

Author: See updates, added new combinations table

> The multiplier modifiers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination - for example, Shift + Control to buy or sell 100 items at a time. All three will be a crazy whopping 50,000x.

Reviewer 2: ```suggestion
The multiplier modifiers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination - for example, Shift + Control to buy or sell 100 items at a time. All three used together will provide a whopping 50,000x multiplier.
```

"will be a" is a little weird phrasing-wise. Not sure if this is better, but it's up to you.

> The multiplier modifiers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination - for example, Shift + Control to buy or sell 100 items at a time. All three used together will provide a whopping 50,000x multiplier.

Reviewer 2: "multiplier modifier" is a little bit of a mouthful. Also, an em-dash is best here because GitHub (unlike ES) actually has unicode support.
```suggestion
These multipliers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination—for example, Shift + Control to buy or sell 100 items at a time. All three used together will provide a whopping 50,000x multiplier.
```

In addition to the multiplier keys, there are various other shortcuts in the outfitter:

> Outfitter: | &nbsp; | Priority 1 | Priority 2 | Priority 3

Reviewer 2: Is there a better way to say this than Priority 1-3? Maybe:
```suggestion
Outfitter: | &nbsp; | First tries: | Then: | And finally:
```

:--- | :--- | :--- | :--- | :---

> Buy | B | Buy* from cargo to ship | Buy* from storage to ship | Buy from outfitter to ship

Reviewer 2: Should this be `Buy*` Or should it just be `Install`?

Author: Yes, I think you're right, that makes it clearer.

> Buy | B | Install\* from cargo to ship | Install\* from storage to ship | Buy from outfitter to ship

Reviewer 2: I don't think the asterisk and footnote are necessary in this case.
```suggestion
Buy | B | Install from cargo to ship | Install from storage to ship | Buy from outfitter to ship
```

Sell | S | Sell from cargo | Sell from storage | Sell from ship
Uninstall | U | Uninstall from ship to cargo | Uninstall, overflow to storage | &nbsp;

> Install | I |Install to ship from cargo | Install to ship from storage | &nbsp;

Reviewer 2: ```suggestion
Install | I | Install to ship from cargo | Install to ship from storage | &nbsp;
```

> Cargo | C | Transfer\* to cargo from planetary storage | Buy from outfitter to planetary storage | &nbsp;

Reviewer 2: ```suggestion
Cargo | C | Transfer to cargo from planetary storage | Buy from outfitter to planetary storage | &nbsp;
```

Author: See latest updates, Buy is just: Buy from outfitter and install on ship
This matches the flow diagram I shared with [#10743](https://github.com/endless-sky/endless-sky/pull/10743)

> Store | Retain in storage from cargo | Uninstall from ship to retain in storage | &nbsp;

Reviewer 2: This line seems wrong somehow? Should be either a shortcut or an empty column?

Reviewer 2: ```suggestion
Store | &nbsp; | Retain in storage from cargo | Uninstall from ship to retain in storage | &nbsp;
```

Author: Actually it needed an "R" not &nbsp;  Fixed.

> \*When you already own it, you won't pay for it again, it'll be installed from cargo or planetary storage rather than paying to install it.

Reviewer 2: Although an explanatory note never hurts...

```suggestion
There is no cost to install outfits, so you won't have to pay to buy and use an outfit when you already have it in your cargo or storage.
```

Shipyard: | &nbsp;
:--- | :---

> B | Buy Ship
> S | Sell Ship with outfits
> R | Sell Ship, Retain outfits in planetary storage

Reviewer 2: ```suggestion
B | Buy ship
S | Sell ship with outfits
R | Sell ship, retain outfits in planetary storage
```

Or you could go all title-case.

(To really pick nits, the comma in the last line should really be replaced with `but`, but I think it's clear enough as-is).

> R |	Sell Ship, Retain outfits in planetary storage

Reviewer 3: What is the logic of the spacing inside this table? The columns don't appear to line up with each other, and the number of spaces is also inconsistent.

Author: no logic, sloppy copy-paste on my part, will fix

Author: they were tabs -- invisible-ish in the editor since it was displaying them as 2-spaces -- so about half of them were only compensating for a single space and got overlooked.

Better?  Wording OK?  Mechanics seems clear?

> R | Sell ship, retain outfits in planetary storage

Reviewer: ```suggestion
R | Sell ship but retain outfits in planetary storage
```

:--- | :---
Buy | +, Return, or Space
Sell | -, Delete, or Backspace
Buy / sell all | Shift + B / S
View commodity price map | Map key

Multipliers: | &nbsp;
:--- | :---
5x | Shift
20x | Control or Command
500x | Option or Alt

> These multipliers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination—for example, Shift + Control to buy or sell 100 items at a time. All three used together will provide a whopping 50,000x multiplier.

Reviewer 4: ```suggestion
These multipliers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination - for example, Shift + Control to buy or sell 100 items at a time. All three used together will provide a whopping 50,000x multiplier.
```
ASCII charset supremacy

:--- | :---
Buy | B
Sell | S
Move/Buy to cargo | C
Move to storage | R 
Install | I
Uninstall | U

> The Buy and Sell shortcuts here are also available in the shipyard.  The Shipyard also supports R for to sell ships and retain their outfits in storage.

Reviewer: ```suggestion
The Buy and Sell shortcuts here are also available in the shipyard. The shipyard also supports R for selling ships and retaining their outfits in storage.
```
